release/_bom.sh

```diff
@@ -67,15 +67,45 @@ function generate_api_jars {
 		rm -f "${name}-${version}.jar"
 	done
 
-	for portal_jar in portal-impl portal-kernel support-tomcat util-bridges util-java util-slf4j util-taglib
-	do
-		_manage_bom_jar "${_BUNDLES_DIR}/tomcat/webapps/ROOT/WEB-INF/shielded-container-lib/${portal_jar}.jar"
-	done
+	if [[ "${_PRODUCT_VERSION}" == "7.3."* ]]
+	then
+		_manage_bom_jar "${_BUNDLES_DIR}/tomcat-9.0.83/lib/ext/portal-kernel.jar"
 
-	find "${_BUNDLES_DIR}/osgi" "${_BUNDLES_DIR}/tomcat/webapps/ROOT/WEB-INF/shielded-container-lib" -name "com.liferay.*.jar" -type f -print0 | while IFS= read -r -d '' module_jar
-	do
-		_manage_bom_jar "${module_jar}"
-	done
+		_manage_bom_jar "${_BUNDLES_DIR}/tomcat-9.0.83/lib/ext/support-tomcat.jar"
+
+		for portal_jar in portal-impl util-bridges util-java util-slf4j util-taglib
+		do
+			_manage_bom_jar "${_BUNDLES_DIR}/tomcat-9.0.83/webapps/ROOT/WEB-INF/lib/${portal_jar}.jar"
+		done
+
+		find "${_BUNDLES_DIR}/osgi" "${_BUNDLES_DIR}/tomcat-9.0.83/lib/ext" "${_BUNDLES_DIR}/tomcat-9.0.83/webapps/ROOT/WEB-INF/lib" -name "com.liferay.*.jar" -type f -print0 | while IFS= read -r -d '' module_jar
```

Comment:
@lucasmiranda0, can you sort the parameters of the commands `find` and `read` alphabetically?